Patch by Daniel Baumann <daniel@debian.org> for librsync >= 0.9.7, which describes
how to use rdiff more verbosely. It updates the man page to remove long time dead
link to Linuxcare Australia as well. For further information, please have a look to
Debian bug IDs #228005, #156457 and #288758.

--- librsync-0.9.7/doc/rdiff.1			2004-02-08 00:17:57.000000000 +0100
+++ librsync-0.9.7/doc/rdiff.1.man_pages	2008-12-20 13:47:45.000000000 +0100
@@ -29,10 +29,17 @@
 .PP
 \fBrdiff\fP [\fIoptions\fP] \fBpatch\fP \fIbasis-file delta-file new-file\fP
 .fi
+.SH USAGE
+You can use \fBrdiff\fP to update files, much like \fBrsync\fP does.
+However, unlike \fBrsync\fP, \fBrdiff\fP puts you in control.  There
+are three steps to updating a file: \fBsignature\fP, \fBdelta\fP, and
+\fBpatch\fP.
 .SH DESCRIPTION
 In every case where a filename must be specified, \- may be used
 instead to mean either standard input or standard output as
-appropriate.
+appropriate.  Be aware that if you do this, you'll need to terminate your
+\fIoptions\fP with \-\- or \fBrdiff\fP will think you are passing it
+an empty option.
 .SH "RETURN VALUE"
 0 for successful completion, 1 for environmental problems (file not
 found, invalid options, IO error, etc), 2 for a corrupt file and 3 for
--- librsync-0.9.7/doc/librsync.3		2004-02-08 00:17:57.000000000 +0100
+++ librsync-0.9.7/doc/librsync.3.man_pages	2008-12-20 13:48:11.000000000 +0100
@@ -53,7 +53,7 @@
 .PP
 .I rdiff and librsync Manual
 .PP
-\fIhttp://rproxy.sourceforge.net/\fP or \fIhttp://linuxcare.com.au/rproxy/\fP.
+\fIhttp://rproxy.sourceforge.net/\fP.
 .PP
 \fIdraft-pool-rsync\fP
 .SH BUGS
